github/repos_test.go

```diff
@@ -555,7 +555,8 @@ func TestRepositoriesService_GetBranchProtection(t *testing.T) {
 					{Slug: String("t"), ID: Int64(4)},
 				},
 			},
-			RequireCodeOwnerReviews: true,
+			RequireCodeOwnerReviews:      true,
+			RequiredApprovingReviewCount: 0,
```

Comment:
Why is the value 3 in the JSON above:

```JSON
"required_approving_review_count":3
```

But `RequiredApprovingReviewCount` is 0 here?